samza-core/src/main/java/org/apache/samza/task/StreamOperatorTask.java

```diff
@@ -133,9 +121,6 @@ public final void window(MessageCollector collector, TaskCoordinator coordinator
 
   @Override
   public void close() throws Exception {
-    if (this.contextManager != null) {
```

Comment:
Where is ApplicationTaskContext started and stopped? In TaskInstance?

Reply:
That is correct.